src/routers/courseItem.ts

```diff
@@ -22,13 +22,45 @@ cItemRouter.get('/:id', async (req, res) => {
 	}
 });
 
+// helper function converts string to number
+//   - should be updated to support fractions
+const numberify = (str: string) => {
+	const regex = /^(\d+\.?\d*)\/(\d+\.?\d*)$/;
+	const match = str.match(regex);
+	if (match) {
+		return Number(match[1]) / Number(match[2]);
+	}
+
+	return Number(str); // Not guranteed to be a number since we don't know what is store in str
+};
+
 cItemRouter.post('/', async (req, res) => {
 	try {
 		const userID = req.header('userID');
+		const body = req.body;
 		if (userID) {
-			await update(req.body, userID);
+			const courseItem = {
+				id: body.id,
+				owner: userID,
+				name: body.name,
+				weight: numberify(body.weight),
+				grade: numberify(body.grade),
+				date: body.date,
+				createdAt: body.createdAt,
+				updatedAt: body.updatedAt,
+				conformToSchema: body.conformToSchema,
+				toDynamo: body.toDynamo,
+				prepareForResponse: body.prepareForResponse,
+				original: body.original,
+				toJSON: body.toJSON,
+				serialize: body.serialize,
+				delete: body.delete,
+				save: body.save,
+				populate: body.populate,
+			};
```

Comment:
@BraidenCutforth is this close to what you had in mind for sanitizing the input JSON? In order for `create` to work it needs ALL the data from `courseItemInterface`. Thoughts?

Reply:
Per conversation we probably need to call this function to create the document so we can remove all keys except the actual data ones.

```suggestion
			const courseItem = new CourseItemModel({
				id: body.id,
				owner: userID,
				name: body.name,
				weight: numberify(body.weight),
				grade: numberify(body.grade),
				date: body.date,
				createdAt: body.createdAt,
				updatedAt: body.updatedAt,
				conformToSchema: body.conformToSchema,
				toDynamo: body.toDynamo,
				prepareForResponse: body.prepareForResponse,
				original: body.original,
				toJSON: body.toJSON,
				serialize: body.serialize,
				delete: body.delete,
				save: body.save,
				populate: body.populate,
			});
```

Reply:
```suggestion
			const courseItem = {
				id: body.id,
				owner: userID,
				name: body.name,
				weight: numberify(body.weight),
				grade: numberify(body.grade),
				date: body.date,
				createdAt: body.createdAt,
				updatedAt: body.updatedAt,
			};
```
Did you mean this?

Reply:
Yup! Sorry about that, forgot to remove the unneeded fields!